server/adapters/aws_redshift.go

```diff
@@ -302,3 +315,36 @@ func (ar *AwsRedshift) recreateNotNullColumnInTransaction(wrappedTx *Transaction
 func (ar *AwsRedshift) Close() error {
 	return ar.dataSourceProxy.Close()
 }
+
+func (ar *AwsRedshift) ValidateWritePermission() error {
```

Comment:
After moving test eventContext creation outside storage, this func can be omitted.

Reply:
Function was removed